target/scripts/build/packages.sh

```diff
@@ -1,10 +1,13 @@
 #!/bin/bash
 
-# -eE :: exit on error (do this in functions as well)
-# -u  :: show (and exit) when using unset variables
+# -eE         :: exit on error (do this in functions as well)
+# -u          :: show (and exit) when using unset variables
 # -o pipefail :: exit on error in pipes
 set -eE -u -o pipefail
 
+# shellcheck source=/dev/null
+source /etc/os-release
```

Comment:
I am also a fan of just include what's needed instead of including all. This seems easy to do at first, but could lead to problems anytime later, when var names otherwise used are overlapping.

Reply:
The we could use `VERSION_CODENAME` instead to at least align with `/etc/os-release`.